content/running_bookwyrm/external-storage.md

```diff
@@ -112,6 +118,19 @@ Then, run the following command:
 
 No output means it should be good.
 
+### Additional Step for Linode Object Storage Users
+
+For Linode, you now need to make an alteration to the `.env` to ensure that the generated links to your storage objects are correct. If you miss this step, all the links to images and static files (like css) will be broken.
+To fix this, you need to now insert the bucket-name into the `AWS_S3_CUSTOM_DOMAIN`, for example if your `AWS_STORAGE_BUCKET_NAME` is `"my-bookwyrm-bucket"`, then set it to:
+
+```
+AWS_S3_CUSTOM_DOMAIN=my-bookwyrm-bucket.cluster-id.linodeobjects.com
+```
+
+*Note*: From this point on, any bw-dev copy or sync commands will place objects into an incorrect location in your object store, so if you need to use them, revert to the previous setting, run and re-enable.
```

Comment:
This seems super messy. I'm wondering if we should add a setting and flag for Linux Object Storage to `bw-dev` instead?

i.e. something like `LOS_S3_STORAGE_BUCKET_NAME` in `.env`, which omits the `my-bookwyrm-bucket` section, and is used in `copy_media_to_s3` and `sync_media_to_s3` if they include a `--los` flag. Then you could do:

```py
# .env
AWS_S3_CUSTOM_DOMAIN=my-bookwyrm-bucket.cluster-id.linodeobjects.com
LOS_S3_STORAGE_BUCKET_NAME=cluster-id.linodeobjects.com
```

and
```sh
./bw-dev copy_media_to_s3 --los
```

Reply:
Hi, 

Yeah, if that's an option it'd be neater for sure. I just thought I'd document the existing procedure for the moment.

cheers,
Damian

Reply:
Yes sorry that probably wasn't worded very helpfully. I meant "Thanks so much for documenting this, and I now realise we should make it easier than the process you have helpfully outlined as the current least-bad option"!

Reply:
:-D Cheers!